lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -18,6 +18,8 @@ module Billing #:nodoc:
     # * The order of the XML elements does matter, make sure to follow the order in
     #   the documentation exactly.
     class CyberSourceGateway < Gateway
+      include ActiveMerchant::Billing::CyberSourceCommon
```

Comment:
As suggested previously, let's wait on making this change until we complete implementation of the REST version

Reply:
I will check :)

Reply:
I think it was your suggestion 😅

Reply:
Ok, thanks. Done!